cypress/e2e/tests/migration/task-manager/manage_columns.test.ts

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright © 2021 the Konveyor Contributors (https://konveyor.io/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+/// <reference types="cypress" />
+
+import {
+    clickByText,
+    login,
+    openManageColumns,
+    restoreColumnsToDefault,
+    selectColumns,
+    validateCheckBoxIsDisabled,
+    validateTextPresence,
+} from "../../../../utils/utils";
+import { TaskManager } from "../../../models/migration/task-manager/task-manager";
+import { button, cancel, save } from "../../../types/constants";
+import { tableHead } from "../../../views/common.view";
+import { TaskManagerTableHeaders } from "../../../views/taskmanager.view";
+
+describe(["@tier3"], "Task manager - table column management validation", function () {
+    const taskManagerDefaultColumns = Object.values(TaskManagerTableHeaders).slice(0, 6);
+    const columnsToSelect = [
+        TaskManagerTableHeaders.pod,
+        TaskManagerTableHeaders.started,
+        TaskManagerTableHeaders.terminated,
+    ];
+
+    before("Login", function () {
+        login();
+    });
+
+    it("Open task manager page - table default columns should be visible", function () {
+        TaskManager.open();
```

Comment:
Could you add the Polarion TC number to the test as a comment ?

Reply:
I don't see it valuable. I prefer not to use fixed numbers (even as a comment)